src/main/kotlin/org/veupathdb/lib/rabbit/jobs/model/JobDispatch.kt

```diff
@@ -38,13 +37,15 @@ data class JobDispatch(
   val jobID:   HashID,
   val body:    JsonNode? = null,
   val type:    String = "",
+  val attemptCount: Int = 0,
   val dispatched: OffsetDateTime = OffsetDateTime.now()
 ) :  JsonSerializable {
   override fun toJson() =
     Json.new<ObjectNode> {
       put(JsonKey.JobID, jobID.string)
       put(JsonKey.Type, type)
       put(JsonKey.Dispatched, dispatched.toString())
+      put(JsonKey.AttemptCount, attemptCount.toString())
```

Comment:
Should the `attemptCount` value be stored as a string or int value?

Reply:
Fixed, sorry I got distracted by git-filter-repo yesterday